src/index.md

```diff
@@ -1,37 +1,142 @@
+  url: "http://apex.apache.org"
+- title: Flink
+  image_url: /images/logo_flink.png
+  url: "http://flink.apache.org"
+- title: Spark
+  image_url: /images/logo_spark.png
+  url: http://spark.apache.org/
+- title: Google Cloud Dataflow
+  image_url: /images/logo_google_cloud.png
+  url: https://cloud.google.com/dataflow/
 
-Apache Beam is:
-* **UNIFIED** - Use a single programming model for both batch and streaming use cases.
-* **PORTABLE** - Execute pipelines on multiple execution environments, including Apache Apex, Apache Flink, Apache Spark, and Google Cloud Dataflow.
-* **EXTENSIBLE** - Write and share new SDKs, IO connectors, and transformation libraries. 
+pillars:
+- title: Unified
+  body: Use a single programming model for both batch and streaming use cases.
+- title: Portable
+  body: Execute pipelines on multiple execution environments.
+- title: Extensible
+  body: Write and share new SDKs, IO connectors, and transformation libraries.
 
-## Get Started
-
-To use Beam for your data processing tasks, start by reading the [Beam Overview]({{ site.baseurl }}/get-started/beam-overview) and performing the steps in the [Quickstart]({{ site.baseurl }}/get-started/quickstart). Then dive into the [Documentation]({{ site.baseurl }}/documentation/) section for in-depth concepts and reference materials for the Beam model, SDKs, and runners.                                    
+cards:
+- quote: "Lorem ipsum dolor sit amit. Qaz qux."
+  name: Person 1
+- quote: "Lorem ipsum dolor sit amit. Qaz qux."
+  name: Person 2
+- quote: "Lorem ipsum dolor sit amit. Qaz qux."
+  name: Person 3
+---
+<div class="hero section">
+  <div class="hero__cols">
+    <div class="hero__cols__col">
+      <div class="hero__cols__col__content">
+        <div class="hero__title">
+          Apache Beam: An advanced unified programming model
+        </div>
+        <div class="hero__subtitle">
+          Implement batch and streaming data processing jobs that run on any execution engine.
+        </div>
+        <div class="hero__ctas">
+          <a class="button button--primary" href="/get-started/beam-overview/">Learn more</a>
+          <a class="button" href="/get-started/quickstart-java/">Quickstart</a>
+        </div>
+      </div>
+    </div>
+    <div class="hero__cols__col">
+      <div class="hero__blog">
+        <div class="hero__blog__title">
+          The latest from the blog
+        </div>
+        <div class="hero__blog__cards">
+          {% for post in site.posts limit:3 %}
+          <a class="hero__blog__cards__card" href="{{ post.url | prepend: site.baseurl }}">
+            <div class="hero__blog__cards__card__title">{{post.title}}</div>
+            <div class="hero__blog__cards__card__date">{{ post.date | date: "%b %-d, %Y" }}</div>
+          </a>
+          {% endfor %}
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
 
-## Contribute 
+<div class="pillars section">
+  <div class="pillars__title">
+    All about beam
```

Comment:
it's hard to imagine how it would look, but would "Apache Beam is" or "Beam is" work better? (kind of goes along with "Works with" in the next block). I'm unsure without actually seeing it though. Maybe someone else has a better suggestion for text here...